CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 0.14.3 - 2025-03-08
+
+- Manually derive `Clone` and `Default` on `WeakLoopHandle`. (#219)
```

Comment:
Looks like https://github.com/Smithay/calloop/pull/216 was also merged after the 0.14.2 tag. 
So adding `WeakLoopHandle` would be the big change in this release.